src/index.tsx

```diff
@@ -30,18 +30,17 @@ const ReactP5WrapperComponent: FC<P5WrapperProps> = ({
   const wrapper = createRef<HTMLDivElement>();
   const [instance, setInstance] = useState<P5Instance>();
 
-  useEffect(() => {
-    instance?.updateWithProps?.(props);
-  }, [props]);
-
   useEffect(() => {
     if (wrapper.current === null) return;
     instance?.remove();
     const canvas = createCanvas(sketch, wrapper.current);
-    canvas.updateWithProps?.(props);
     setInstance(canvas);
   }, [sketch, wrapper.current]);
 
+  useEffect(() => {
+    instance?.updateWithProps?.(props);
```

Comment:
Since `useEffect` calls run in order, we don't need the second call that used to exist which means we have one line of code less and deduplicated logic which works for me any day of the week 😉!

Reply:
Makes sense, thanks for this!